dora/tests/integration/src/test/java/alluxio/client/fs/DoraFileSystemIntegrationTest.java

```diff
@@ -94,9 +100,25 @@ public final class DoraFileSystemIntegrationTest extends BaseIntegrationTest {
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
   private AmazonS3 mS3Client = null;
+  @Rule
+  public TemporaryFolder mFolder = new TemporaryFolder();
 
   @Before
   public void before() throws Exception {
+    File file = mFolder.newFile();
```

Comment:
remove this?

Reply:
We need a static file to provide worker memberships. I don't think the test will pass if I remove the file.

Reply:
Does it still work if you use `WORKER_MEMBERSHIP_MANAGER_TYPE = MASTER`? So workers register to the master.

Reply:
No. Because we have modify the mechanism of worker registration, if we use `WORKER_MEMBERSHIP_MANAGER_TYPE = MASTER`, the worker will not register to master correctly.

Reply:
Now the temporary file and directory are removed.